PyDSS/registry.py

```diff
@@ -214,6 +215,7 @@ def show_controllers(self):
                     filename = controller["filename"]
                     print(f"  {name}:  {filename}")
             else:
+                ...
```

Comment:
stale?

Reply:
I dont think so. it is being used in CLI

Reply:
You seem to have added the `…`. Shouldn’t it be removed? It doesn’t do anything.

Reply:
removed